### PR TITLE
Add webdriver test to noScript exceptions on compact bravery panel and apply .changeSetting

### PR DESCRIPTION
**Pull request by luixxiul**
Follow-up to #8837

Because `does not apply exceptions from private tabs to regular tabs` is already complex somewhat, I'm adding another test to cover #8837 on compact bravery panel.

Also I applied `this.app.client.changeSetting` to the other tests with https://github.com/brave/browser-laptop/pull/9075/commits/17c2f041571750816ab4fc92fa9de68885b009b6

Test Plan:

    npm run test -- --grep='does not apply exceptions from private tabs to regular tabs on compact panel'
    npm run test -- --grep='Bravery Panel'

Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [x] Added/updated tests for this change (for new code or code which already has tests).
- [x] Ran `git rebase -i` to squash commits (if needed).

Reviewer Checklist:

Tests

- [ ] Adequate test coverage exists to prevent regressions
- [ ] Tests should be independent and work correctly when run individually or as a suite [ref](https://github.com/brave/browser-laptop/wiki/Code-Guidelines#test-dependencies)
- [ ] New files have MPL2 license header

### CODE REVIEW

File: test/bravery-components/braveryPanelTest.js [hunk 1]
```diff
@@ -648,6 +648,57 @@ describe('Bravery Panel', function () {
...
     })
+    it('does not apply exceptions from private tabs to regular tabs on compact panel', function * () {
+      const url = Brave.server.url('scriptBlock.html')
+      yield this.app.client
+        .tabByIndex(0)
+        .loadUrl(prefsShieldsUrl)
+        .waitForVisible(compactBraveryPanelSwitch)
+        .click(compactBraveryPanelSwitch)
+        .windowByUrl(Brave.browserWindowUrl)
+        .keys(Brave.keys.ESCAPE)
```
Comment (diracdeltas) [inline test/bravery-components/braveryPanelTest.js]: i think you can avoid the above by just using `this.app.client.changeSetting`

Comment (diracdeltas) [inline test/bravery-components/braveryPanelTest.js]: otherwise lgtm

Comment (luixxiul) [inline test/bravery-components/braveryPanelTest.js]: > this.app.client.changeSetting

ya that's the solution I have been looking for. thanks for the tip.

Comment (luixxiul) [inline test/bravery-components/braveryPanelTest.js]: Fixed with https://github.com/brave/browser-laptop/pull/9075/commits/17c2f041571750816ab4fc92fa9de68885b009b6#diff-f94f0ea4252431046502bbe75728ba1fR620

**Pull request merged**